### ISSUE TITLE
Use brew command directly to kick the build in OSBS

### ISSUE THREAD

**Issue by dbecvarik** [cekit/cekit#183]
Using `brew` directly will give us control over the output and the build process. We can wait and return the task/build information in the format we want.

**Comment by goldmann** [comment]
Notes:

* To get url of remote: `git remote get-url origin`, URL
* To get latest hash: `git rev-parse HEAD`, HASH

```groovy
def uri = new URI(URL)
// This is how we should generate the 'src' parameter to brew call
println("git://${uri.host}${uri.path}/#origin/#{HASH}")
```

**Comment by goldmann** [comment]
Implemented in #397.
